pkg/authorization/delegated/authorizer.go

```diff
@@ -30,14 +30,14 @@ import (
 	"k8s.io/klog/v2"
 )
 
-type DelegatedAuthorizerFactory func(clusterName logicalcluster.Name, client kubeclient.ClusterInterface) (authorizer.Authorizer, error)
+type DelegatedAuthorizerFactory func(clusterName logicalcluster.Name, client kubeclient.Interface) (authorizer.Authorizer, error)
 
 // NewDelegatedAuthorizer returns a new authorizer for use in e.g. admission plugins that delegates
 // to the kube API server via SubjectAccessReview.
-func NewDelegatedAuthorizer(clusterName logicalcluster.Name, client kubeclient.ClusterInterface) (authorizer.Authorizer, error) {
+func NewDelegatedAuthorizer(clusterName logicalcluster.Name, client kubeclient.Interface) (authorizer.Authorizer, error) {
 	delegatingAuthorizerConfig := &authorizerfactory.DelegatingAuthorizerConfig{
 		SubjectAccessReviewClient: &clusterAwareAuthorizationV1Client{
-			AuthorizationV1Interface: client.Cluster(clusterName).AuthorizationV1(),
```

Comment:
Where did this go?

Reply:
Passing a unscoped client here: https://github.com/kcp-dev/kcp/pull/1753/files#diff-32c06ebe8bfea0a512b39070c4654c7751769279e2b921a57698df08326f9887R40